src/main/java/frc/robot/subsystems/Claw.java

```diff
@@ -57,4 +60,13 @@ public void toggleClaw() {
   public void periodic() {
     // This method will be called once per scheduler run
   }
+  private static void configDriveMotor(WPI_TalonFX driveMotor){
+    driveMotor.configFactoryDefault(); // resets the motor to its factory default settings. 
+    driveMotor.setNeutralMode(NeutralMode.Brake); //sets neutral mode of robot to break.
+    driveMotor.config_KP(1, p);
+    driveMotor.config_KI(0, i);
+    driveMotor.config_KD(0, d);
+    driveMotor.config_KF(0, f);
```

Comment:
The variables p, i, d, and f were just put there as an example.  They should be changed for the actual numbers, 1, 0, 0, and 0.
Also, the real method call should be spelled with a lowercase k.